packages/office-ui-fabric-react/src/common/isConformant.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/no-extraneous-dependencies -- this file is for testing
```

Comment:
Probably better to remove these individual disables and centrally handle this in `packages/eslint-plugin/src/utils/configHelpers.js`: in the `testFiles` array add a line `'**/common/isConformant.{ts,tsx}'`